.travis.yml

```diff
@@ -19,20 +19,22 @@ matrix:
         - php: 7.1
           env: SYMFONY_VERSION=3.3.* TEST_COVERAGE=true
         - php: 7.1
-          env: GRAPHQLPHP_VERSION=0.10.0
+          env: SYMFONY_VERSION=3.4.* GRAPHQLPHP_VERSION=0.10.0 DEPENDENCIES=dev
         - php: hhvm
+          dist: trusty
+          group: edge
         - php: nightly
-          env: COMPOSER_UPDATE_FLAGS=--ignore-platform-reqs
+          env: SYMFONY_VERSION=4.0.* DEPENDENCIES=dev COMPOSER_UPDATE_FLAGS=--ignore-platform-reqs
```

Comment:
Let's make a build 7.1 with symfony 4 and leave the nightly more accordingly to our composer.json?

Reply:
yes, I will make the change